server/src/main/java/org/elasticsearch/index/fieldvisitor/FieldsVisitor.java

```diff
@@ -60,12 +59,6 @@ public Status needsField(FieldInfo fieldInfo) {
         if (requiredFields.remove(fieldInfo.name)) {
             return Status.YES;
         }
-        // Always load _ignored to be explicit about ignored fields
-        // This works because _ignored is added as the first metadata mapper,
-        // so its stored fields always appear first in the list.
-        if (IgnoredFieldMapper.NAME.equals(fieldInfo.name)) {
-            return Status.YES;
-        }
```

Comment:
the fact that tests don't fail with this removed is a red flag? It's good for fetch in search because `_ignored` retrieval goes via value fetcher even when stored. But the get API should still need this?